.github/workflows/mattermost-ziti-webhook.yml

```diff
@@ -1,25 +1,26 @@
-name: mattermost-ziti-webhook
-on: 
-  create:
-  delete:
+name: ziti-mattermost-action-py
+on:
   issues:
   issue_comment:
   pull_request_review:
   pull_request_review_comment:
   pull_request:
+    types: [opened, reopened]
```

Comment:
Why opened and reopened? (Also I am making this comment just because I want to see what it looks like in mm)